lib/localstorage.provider.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const common = require('metarhia-common');
+const localStorage = require('localStorage');
+const { StorageProvider } = require('./provider');
+const { MemoryCursor } = require('./memory.cursor');
+
+function LocalstorageProvider() {}
+
+common.inherits(LocalstorageProvider, StorageProvider);
+
+LocalstorageProvider.ID_LABEL = '_LocalstorageProviderId';
+LocalstorageProvider.IDS_LABEL = '_LocalstorageProviderIds';
+LocalstorageProvider.ID_ITEM_LABEL = '_LocalstorageProvider_item_';
```

Comment:
Please add comments to describe this labels

Reply:
Thank you, I've not noticed that IDS_LABEL isn't even used.